src/abstract/Module.ts

```diff
@@ -1,7 +1,7 @@
-import { isElement, isUndefined } from 'underscore';
-import { Collection, View } from '../common';
-import EditorModel from '../editor/model/Editor';
-import { createId, isDef } from '../utils/mixins';
+import { isElement, isUndefined } from "underscore";
```

Comment:
Aren't you able to fix the issue with quotes so we can suppress all the warnings in the PR?
In case you can't I'd ask you to run `npm run lint -- --fix`

Reply:
I am including `ts` files on lint-staged to auto-fix this kind of lining issue before committing in the future. 
https://github.com/artf/grapesjs/pull/4375